core/resource/resource.go

```diff
@@ -233,7 +233,7 @@ func (r *Resource) MarkSkipped() error {
 }
 
 func (r *Resource) MarkToCreate() error {
-	if r.status == StatusValidationSuccess || r.status == StatusCreateFailure {
+	if r.status == StatusValidationSuccess {
```

Comment:
What is the reason of changing the requirement to transition status into `to_create` only status `success`? I think it's a bit different than the last discussion.

Reply:
The current implementation was wrong. Let's have a call if you have some confusion. Only the resources in validation success can be marked as to_create. It will not be done for existing resources.